spec/lib/rbac/filterer_spec.rb

```diff
@@ -1286,56 +1287,143 @@ def get_rbac_results_for_and_expect_objects(klass, expected_objects)
         end
       end
 
-      it "returns users from current user's groups" do
-        other_user.miq_groups << group
-        get_rbac_results_for_and_expect_objects(User, [user, other_user])
-      end
+      context "with other users in the group" do
```

Comment:
I think the fact that other users are in the group is a given/base case and should not be a context. (starts to make it indention happy)

Reply:
yeah I was struggling here - it's important to have multiple users to show which ones are or aren't coming back.  These tests are bigger than the "self" case.

Reply:
I decided to keep it.  Reason it that otherwise you just have these floating `it`s and it's hard to tell why they aren't in a context and what makes them special.